opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -160,6 +160,13 @@ def get_description(self) -> str:
         return "AlwaysOnSampler"
 
 
+ALWAYS_OFF = StaticSampler(Decision.DROP)
+"""Sampler that never samples spans, regardless of the parent span's sampling decision."""
+
+ALWAYS_ON = StaticSampler(Decision.RECORD_AND_SAMPLE)
+"""Sampler that always samples spans, regardless of the parent span's sampling decision."""
```

Comment:
Do you need to rebase? These are already in the file
https://github.com/open-telemetry/opentelemetry-python/blob/d2028958d75b6399af1331c261f024993a4da476/opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py#L265-L269

Reply:
Really? I simply just moved them from the bottom to the top because `ParentBased` needs access to them.